cdi/src/main/java/org/hibernate/validator/internal/cdi/ValidatorFactoryBean.java

```diff
@@ -51,7 +53,7 @@
 
 	public ValidatorFactoryBean(BeanManager beanManager, ValidationProviderHelper validationProviderHelper) {
 		this.beanManager = beanManager;
-		this.destructibleResources = newHashSet( 4 );
+		this.destructibleResources = newHashSet( 5 );
```

Comment:
You lost your right to make this sort of comment when you voted against my PR to adapt CollectionHelper to avoid the resize :) - see https://github.com/hibernate/hibernate-ogm/pull/683.

I really think we should do it once and for all and be done with it.